extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -106,7 +113,14 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
         .getVisibilityState();
     const adTestOn = this.element.getAttribute('data-adtest') ||
         isInManualExperiment(this.element);
-    const format = `${slotRect.width}x${slotRect.height}`;
+    const width = slotRect.width;
```

Comment:
Status of what? I've changed how the format is calculated for DoubleClick here: https://github.com/ampproject/amphtml/pull/7334.